### PR TITLE
Added Routes to service

### PR DESCRIPTION
**Pull request by kimberlyjk**
Just like the title suggests, I added routes to my service so that the page changes as the campId changes. 

I also added a test to make sure the data was actually changing. __PLEASE NOTE: i added new npm pkgs to this PR and so in order to run the tests, you will need to make an `npm install` call__

### CODE REVIEW

File: babel.config.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
     [
     "@babel/preset-env",
     {
+      "targets": {
```
Comment (djian001) [inline babel.config.json:7]: what does this do :O

File: client/overview.jsx [hunk 2]
```diff
@@ -10,12 +10,12 @@ import Essentials from './Components/essentials.jsx';
 import Amenities from './Components/amenities.jsx';
 import Details from './Components/details.jsx';
 
-
 class Overview extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+    let query = new URLSearchParams(document.location.search);
```
Comment (djian001) [inline client/overview.jsx:16]: interesting... I tried to use new URLSearchParams too...

File: testing/api.spec.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const database = require('../database/index.js');
 const axios = require('axios');
 const mongoose = require('mongoose');
+const regeneratorRuntime = require("regenerator-runtime");
```
Comment (djian001) [inline testing/api.spec.js:4]: What is this package used for?

File: testing/route.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+
+jest.mock('axios');
+describe('routes', () => {
+  test('should pull up specific infor for campId routes', async () => {
```
Comment (djian001) [inline testing/route.spec.js:5]: wow you're so on top of your testing 🔥

**Review by djian001 — APPROVED** [review]

File: client/overview.jsx (hunk 2 above)
Comment (kimberlyjk) [reply client/overview.jsx:16]: i can try to help you out if you want! I used the console on google first to figure out if i could key into what i needed...

File: testing/api.spec.js (hunk 3 above)
Comment (kimberlyjk) [reply testing/api.spec.js:4]: this is a jest requirement.. i was using babel-polyfill but then i had stuff wrong with my package.lock.json and when i "fixed" it, i was getting this error thrown on jest

**Pull request merged**